### DISCUSSION TITLE
Heading title with outliner in editor mode.

### DISCUSSION THREAD

**Discussion by connoryang331** [vslinko/obsidian-outliner#171] [Archive]
I'm not sure this feature if it can be implemented by plugin. 

Is possible implement Heading title with outliner in editor mode. ? 

Currently :

Editor mode.:
![image](https://user-images.githubusercontent.com/81000027/124528157-14996a00-de3a-11eb-80c9-3376c37b4b36.png)

Preview mode:
![image](https://user-images.githubusercontent.com/81000027/124528262-5fb37d00-de3a-11eb-9df1-2a5274f3d6b1.png)

Hope result : 

Heading title with outliner in editor mode.

in dynalist ：
![image](https://user-images.githubusercontent.com/81000027/124528402-ad2fea00-de3a-11eb-9180-490098afbd80.png)

**Comment by vslinko** [comment]
@Connor-yang96 Hi! Thank you for your contribution!

I think it's probably possible, but I'm sure it would be hard to implement.
I'm unsure if it will be done, but I'll leave it on the list of ideas.

**Reply by connoryang331** [reply]
Doesn't matter if implement. Bcz "WYSIWYG editor " in Short term.
